File: Web/RMA-Web-Shell/projects/shared-components-lib/src/lib/icd10-code-filter-dialog/icd10-code-model.ts
import { InjurySeverityTypeEnum } from "projects/claimcare/src/app/claim-manager/shared/enums/injury-severity-type-enum";
import { BodySideAffectedTypeEnum } from "projects/shared-models-lib/src/lib/enums/body-side-affected-type-enum";

export class ICD10CodeModel {
    eventType: number;
    icd10CodeId: number;
    icd10DiagnosticGroupId: number;
    icd10CategoryId: number;
    icd10SubCategoryId: number;
    icd10Code: string;
    icd10DiagnosticGroupCode: string;
    icd10DiagnosticGroupDescription: string;
    icd10CategoryCode: string;
    icd10CategoryDescription: string;
    icd10SubCategoryCode: string;
    icd10SubCategoryDescription: string;
    icd10CodeDescription: string;
    displayValue: string;
    isActive: boolean;
    bodySideAffected: BodySideAffectedTypeEnum;
    bodySideComment: string;
    severity: InjurySeverityTypeEnum;
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/ICD10CodeModel.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class ICD10CodeModel
    {
        public EventTypeEnum? EventType { get; set; }
        public int Icd10DiagnosticGroupId { get; set; }
        public int Icd10CategoryId { get; set; }
        public int Icd10SubCategoryId { get; set; }
        public int Icd10CodeId { get; set; }
        public string Icd10Code { get; set; }
        public int ProductId { get; set; }
        public string Icd10DiagnosticGroupCode { get; set; }
        public string Icd10DiagnosticGroupDescription { get; set; }
        public string Icd10CategoryCode { get; set; }
        public string Icd10CategoryDescription { get; set; }
        public string Icd10SubCategoryCode { get; set; }
        public string Icd10SubCategoryDescription { get; set; }
        public string Icd10CodeDescription { get; set; }
        public string DisplayValue { get; set; }
        public bool IsActive { get; set; }
    }
}
